eng/source-build-patches/0015-Use-1.0-version-of-ref-assemblies.patch

```diff
@@ -0,0 +1,25 @@
+From a1d1e603e5ffc5cc0402725819ea3a29e71e6c7b Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Sat, 5 Sep 2020 02:46:44 -0500
+Subject: [PATCH] Use 1.0 version of ref assemblies
+
+---
+ src/libraries/pkg/test/frameworkSettings/net/settings.targets | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/libraries/pkg/test/frameworkSettings/net/settings.targets b/src/libraries/pkg/test/frameworkSettings/net/settings.targets
+index e4e8ff23d68..edfeff17503 100644
+--- a/src/libraries/pkg/test/frameworkSettings/net/settings.targets
++++ b/src/libraries/pkg/test/frameworkSettings/net/settings.targets
+@@ -8,7 +8,7 @@
+   <ItemGroup>
+     <!-- For .NET Framework we reference this package which has the same layout as the SDK, with this package,
+     tests can run in any machine without having to install .NET Framework. -->
+-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2" />
++    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
```

Comment:
This patch will likely not work anymore as well after https://github.com/dotnet/runtime/pull/53936 goes in.

Reply:
This patch has been removed, compliments of you! Thanks